Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/MainActivity.java

```diff
@@ -124,7 +127,9 @@ private void replaceFragment(String title) {
     if (fragment == null) {
       // Initialize the fragment based on tag
       if (title.equals(getResources().getString(R.string.str_agenda))) {
-        fragment = new AgendaFragment();
+        fragment = AgendaFragment.Companion.newInstance(false);
+      } else if (title.equals(getResources().getString(R.string.str_my_schedule))) {
+        fragment = AgendaFragment.Companion.newInstance(true);
```

Comment:
might be better for two distinctly named methods for my agenda vs. not rather than a boolean arg here